app/services/location-service.js

```diff
@@ -21,6 +23,20 @@ export default ApiBaseService.extend(NavigationAwareness, {
     this.getWithDefault("onLocationSelected", _.noop)(null);
   },
 
+  async movePackage(pkg, opts = {}) {
+    const { from, to, quantity } = opts;
+
+    const payload = await this.PUT(`/packages/${pkg.get("id")}/move`, {
```

Comment:
Is there any scenario where we need to show an error to users if any move operation fails?

Reply:
There are, but not in the service. An exception coming from the service is a perfectly valid outcome.

Whoever calls the service has to handle the error, that would usually be the page.
Right now it's letting the default error handler show the popup, but I do have an improvement in mind regarding that : )